deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -279,6 +279,16 @@ spec:
                           the taint. By default, it is not set, which means tolerate the taint forever
                           (do not evict). Zero and negative values will be treated as 0 (evict
                           immediately) by the system.
+            pxRepo:
```

Comment:
Why do we need this to be a section in the StorageCluster spec. I think we can always keep this enabled and running. Is there is need to expose this to the user?

Reply:
after some considerations, i think it's better to keep it, we can make default as enabled. there are cases users may want to disable it.
- the image is too big
- there is unexpected issue in the feature